app.js

```diff
@@ -0,0 +1,40 @@
+const express = require("express")
+const cors = require("cors")
+const bodyParser = require('body-parser')
+const School = require("./School.js")
+
+const app = express()
+app.use(cors())
+app.use(bodyParser.urlencoded({extended: false}))
+app.use(bodyParser.json())
+
+const port = 3000
+
+let mySchool = new School()
+
+const showAllCourses = (req, res) => {
+    res.json({
+        status: 200,
+        message: "Recieved all courses",
+        allCourses: mySchool.courses
+    })
+}
+
+app.get("/course", showAllCourses)
+
+
+app.post("/school", (req, res) => {
+    mySchool.enrollStudent(req.body.name, req.body.age, req.body.city, req.body.grade)
```

Comment:
The enrollStudent function is expecting 2 arguments - the name of the course and student info as an object. You can pass the name of the course as a parameter, and student as `req.body` . The parameter would come from the front end. But you didn't write the function there.